mm2src/mm2_bitcoin/keys/src/address.rs

```diff
@@ -216,7 +216,7 @@ impl FromStr for Address {
 }
 
 impl From<&'static str> for Address {
-    fn from(s: &'static str) -> Self { s.parse().unwrap() }
```

Comment:
Ideally this should be conditionally compiled for tests only.

Reply:
Yes, here is a PR for refactoring this struct and a discussion on this unwrap in it: https://github.com/KomodoPlatform/komodo-defi-framework/pull/1960#discussion_r1321119993